### PR TITLE
RDBC-285 Add test for zero boosting

### PR DESCRIPTION
**Pull request by ayende**

### CODE REVIEW

File: test/SlowTests/Issues/RDBC-285.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+namespace SlowTests.Issues
+{
+    public class RDBC_285 : RavenTestBase
+    {
+        [Fact]
+        public void CanQueryWithBoostEqZero()
+        {
+            using (var store = GetDocumentStore())
+            {
+                store.Maintenance.Send(new CreateSampleDataOperation());
```
Comment (ppekrol) [inline test/SlowTests/Issues/RDBC-285.cs:20]: Test failed. We need to wait for indexing after this I think?

Comment (ayende) [reply test/SlowTests/Issues/RDBC-285.cs:20]: Yep
